civilcode.shrc

```diff
@@ -4,9 +4,19 @@ alias dea="docker-compose exec application"
 alias deat="docker-compose exec -e MIX_ENV=test application"
 alias deam="docker-compose exec application mix"
 alias deatm="docker-compose exec -e MIX_ENV=test application mix"
+alias dcs="docker-compose stop"
+alias dcu="docker-compose up -d"
 
 alias elixir-format="watchman -- trigger ./ format '**/*.ex' '**/*.exs' '**/*.default' -- mix format"
 
+function deamtw() {
+	docker-compose exec application mix test.watch
+}
```

Comment:
Why does this need to be an alias?

Reply:
It does not. Lazy I guess. Will convert to an alias